.github/workflows/release.yml

```diff
@@ -355,17 +355,20 @@ jobs:
           script: |
             const { owner, repo } = context.repo
             await github.repos.updateRelease({ owner, repo, release_id: ${{ needs.set-release-id.outputs.release_id }}, prerelease: false })
-      - name: Update dependents
+      - name: 'Update dependents'
         env:
-          JENKINS_DEVOPS_TOKEN: ${{ secrets.JENKINS_DEVOPS_TOKEN }}
+          GITHUB_TOKEN: ${{ secrets.JENKINS_GITHUB_PAT }}
           VERSION: ${{ needs.macos-build.outputs.version }}
         run: |
-          curl --fail 'https://ci.runtimeverification.com/jenkins/buildByToken/buildWithParameters' \
-              --data job=Devops/master \
-              --data token=$JENKINS_DEVOPS_TOKEN \
-              --data UPDATE_DEPS=true \
-              --data UPDATE_DEPS_REPO=runtimeverification/k \
-              --data UPDATE_DEPS_VERSION=v$VERSION
+          set -x
+          version="${GITHUB_SHA}"
```

Comment:
```suggestion
          set -x
          version="${VERSION}"
```

Reply:
You can also remove the `v` from eth line 371 in `"version":"'v${VERSION}'"`, and add it here instead, if you want.